tests/hirte_test/config.py

```diff
@@ -71,7 +71,7 @@ def __init__(
             manager_host: str = "",
             manager_port: str = "8420",
             manager_address: str = "",
-            heartbeat_interval: str = "",
+            heartbeat_interval: str = "2000",
```

Comment:
Shouldn't this be AGENT_HEARTBEAT_INTERVAL_MSEC  as a string?

Reply:
Morning @rhatdan,
 Not sure if I follow, should we create in the python test files a constant like in the C?

Reply:
Should not review code early in morning, I did not notice one was in C and the other Python.
Never mind.